modules/islandora_videojs_dash/README.md

```diff
@@ -0,0 +1,39 @@
+# Islandora Video.js MPEG-Dash support
+
+## Introduction
+
+Adds MPEG-Dash support to the islandora_videojs module via the videojs-contrib-dash.js plugin.
+
+## Requirements
+
+* [Islandora Video.js](https://github.com/islandora/islandora_videojs)
+* [Video.js](https://github.com/videojs/video.js/releases/download/v5.10.2/video-js-5.10.2.zip)
```

Comment:
Not really needed here, as it's for this that we're dependent on `islandora_videojs`, no?